.github/workflows/android.yml

```diff
@@ -12,10 +12,16 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+        
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.8.0
```

Comment:
[Latest version seems to be 0.9.1](https://github.com/marketplace/actions/cancel-workflow-action), any reason to not use that one?
If yes, would you mind adding a comment?

Reply:
@nicolas-raoul sorry i missed out that, can you review now ? i have updated